website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -40,68 +48,301 @@ Authentication uses a dbt Cloud [service account tokens](/docs/dbt-cloud-apis/se
 {"Authorization": "Bearer <SERVICE TOKEN>"}
 ```
 
-Each GQL request also comes with a dbt Cloud environmentId. The API uses both the service token in the header and environmentId for authentication.
+Each GQL request also requires a dbt Cloud `environmentId`. The API uses both the service token in the header and environmentId for authentication.
 
-### Metric metadata calls
+### Metadata calls
 
-Use the following example calls to provide you with an idea of the types of commands you can use:
+**Fetch Data Platform Dialect** 
+
+In some cases in your application, it may be useful to know the dialect/data platform that's being used under the hood for the dbt Semantic Layer connection (e.g., if you are building where filters from a GUI rather than user-inputted SQL) The GraphQL API has an easy way to fetch this via the following query: 
+
+```graphql
+{
+    environmentInfo(environmentId: BigInt!) {
+            dialect
+    }
+}
+```
 
 **Fetch available metrics**
 
 ```graphql
-metrics(environmentId: Int!): [Metric!]!
+metrics(environmentId: BigInt!): [Metric!]!
 ```
 
 **Fetch available dimensions for metrics**
 
 ```graphql
 dimensions(
-environmentId: Int!
-metrics: [String!]!
+  environmentId: BigInt!
+  metrics: [String!]!
 ): [Dimension!]!
 ```
 
-**Fetch available time granularities given metrics**
+**Fetch available granularities given metrics**
+
+Note: This call for `queryableGranularities` returns only queryable granularities for metric time - the primary time dimension across all metrics selected.
 
 ```graphql
 queryableGranularities(
-environmentId: Int!
-metrics: [String!]!
+  environmentId: BigInt!
+  metrics: [String!]!
 ): [TimeGranularity!]!
 ```
 
+You can also get queryable granularities for all other dimensions using the `dimensions` call:
```

Comment:
@courtneyholcomb  do you think you can add a "fetch what agg_time_dimension" makes up metric time before or after this example?

Reply:
Yep I'll add a commit to this PR!

Reply:
Done!